packages/core/src/utils.ts

```diff
@@ -401,10 +401,9 @@ export const claimExecutorPayment = async (
 ) => {
   // The amount to withdraw is the minimum of the executor's debt and the ChugSplashManager's
   // balance.
-  const withdrawAmount = Math.min(
-    await ChugSplashManager.executorDebt(executor.address),
-    await ChugSplashManager.getBalance()
-  )
+  const debt = await ChugSplashManager.executorDebt(executor.address)
+  const balance = await executor.provider.getBalance(ChugSplashManager.address)
+  const withdrawAmount = debt < balance ? debt : balance
```

Comment:
Unfortunately, math.min does not work on BigNumbers.

Reply:
Ah i see. I think we should use the [`lt` operator](https://docs.ethers.org/v5/api/utils/bignumber/#BigNumber--BigNumber--methods--comparison-and-equivalence) then since we're working with BigNumbers